Drive/rover_drive_system.hpp

```diff
@@ -42,47 +45,31 @@ class RoverDriveSystem : public sjsu::common::RoverSystem
   /// Initializes wheels and sets rover to operational starting mode (spin)
   void Initialize()
   {
-    try
-    {
-      sjsu::LogInfo("Initializing drive system...");
-      mc_data.is_operational = 1;
+    sjsu::LogInfo("Initializing drive system...");
+    mc_data.is_operational = 1;
 
-      left_wheel_.Initialize();
-      right_wheel_.Initialize();
-      back_wheel_.Initialize();
-      HomeWheels();
-    }
-    catch (const std::exception & e)
-    {
-      sjsu::LogError("Error initializing!");
-      throw e;
-    }
+    left_wheel_.Initialize();
+    right_wheel_.Initialize();
+    back_wheel_.Initialize();
+    HomeWheels();
```

Comment:
I thought we found this line to be a bug. Should be 
```suggestion
    SetSpinMode();
```

Reply:
That and we should probably initialize all the variables within the classes to prevent errors like this from reoccurring but that can be another issue / pr

Reply:
> That and we should probably initialize all the variables within the classes to prevent errors like this from reoccurring but that can be another issue / pr

will do this in another pr then